### PR TITLE
Completed solution

### PR DESCRIPTION
**Pull request by sergey-lipikhin**

### CODE REVIEW

**Review by Stepan-TS — APPROVED** [review]
good job!

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -29,6 +29,15 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const total = days * 40;
+
+  if (days >= 7) {
+    return total - 50;
+  } else if (days >= 3) {
+    return total - 20;
+  }
```
Comment (Andr1yK) [inline src/calculateRentalCost.js:38]: what is 20/50/40?

**Review by Andr1yK — CHANGES_REQUESTED** [review]
check by checklist

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
```
Comment (Andr1yK) [inline src/calculateRentalCost.js:31]: do u need this comment?

**Review by Andr1yK — APPROVED** [review]

**Pull request closed without merging**